src/utils/format-date.js

```diff
@@ -62,3 +69,8 @@ export function getBeginDate(periodString) {
   }
   return beginDate.unix()
 }
+
+export function dateHasPassed(unitTimestamp) {
```

Comment:
Was this supposed to be `unixTimestamp`?

Reply:
yea, typo will fix